src/watertap_contrib/seto/costing/solar/flat_plate_surrogate.py

```diff
@@ -0,0 +1,185 @@
+import pyomo.environ as pyo
+from watertap.costing.util import register_costing_parameter_block
+from watertap_contrib.seto.costing.util import (
+    make_capital_cost_var,
+    make_fixed_operating_cost_var,
+    make_variable_operating_cost_var,
+)
+
+
+def build_flat_plate_surrogate_cost_param_block(blk):
+
+    costing = blk.parent_block()
+
+    blk.cost_per_area_collector = pyo.Var(
+        initialize=600,
+        units=costing.base_currency / pyo.units.m2,
+        bounds=(0, None),
+        doc="Cost per area for solar collector",
+    )
+
+    blk.cost_per_volume_storage = pyo.Var(
+        initialize=2000,
+        units=costing.base_currency / pyo.units.m3,
+        bounds=(0, None),
+        doc="Cost per volume for thermal storage",
+    )
+
+    blk.balance_of_system_cost = pyo.Var(
+        initialize=1000,
+        units=pyo.units.dimensionless,
+        bounds=(0, None),
+        doc="Balance of system cost",
+    )
+
+    blk.installation_cost = pyo.Var(
+        initialize=2500,
+        units=pyo.units.dimensionless,
+        bounds=(0, None),
+        doc="Installation cost",
+    )
+
+    blk.contingency_frac_direct_cost = pyo.Var(
+        initialize=0,
+        units=pyo.units.dimensionless,
+        bounds=(0, 1),
+        doc="Fraction of direct costs for contingency",
+    )
+
+    blk.indirect_frac_direct_cost = pyo.Var(
+        initialize=0,
+        units=pyo.units.dimensionless,
+        bounds=(0, 1),
+        doc="Fraction of direct costs, including contingency, for indirect costs",
+    )
+
+    blk.tax_frac_direct_cost = pyo.Var(
+        initialize=1,
+        units=pyo.units.dimensionless,
+        bounds=(0, 1),
+        doc="Fraction of direct costs applicable for sales tax",
+    )
+
+    blk.fixed_operating_by_capacity = pyo.Var(
+        initialize=16,
+        units=costing.base_currency / (pyo.units.kW * costing.base_period),
+        bounds=(0, None),
+        doc="Fixed operating cost of flat plate plant per kW capacity",
+    )
+
+    blk.variable_operating_by_generation = pyo.Var(
+        initialize=0,
+        units=costing.base_currency / (pyo.units.MWh * costing.base_period),
+        bounds=(0, None),
+        doc="Variable operating cost of flat plate plant per MWh generated",
+    )
+
+    blk.fix_all_vars()
```

Comment:
Actually, this line is not necessary, as it is accounted for in the decorator below.

Reply:
Thanks. Removed.